__tests__/app.test.js

```diff
@@ -93,6 +93,73 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api/articles/:article_id", () => {
+  describe("GET", () => {
+    describe("Happy paths", () => {
+      test("Returns an object with a key: article", () => {
```

Comment:
Test description could be more descriptive

Reply:
Between the describes and the test I can't think of how this could be more descriptive...? Rayhaan had actually noted that test is a little irrelevant so I have removed these from my current code.